Runtime/JSValue.cs

```diff
@@ -249,6 +249,17 @@ public static unsafe JSValue CreateBigInt(int signBit, ReadOnlySpan<ulong> words
     public static implicit operator JSValue(ulong value) => CreateNumber(value);
     public static implicit operator JSValue(float value) => CreateNumber(value);
     public static implicit operator JSValue(double value) => CreateNumber(value);
+    public static implicit operator JSValue(bool? value) => value.HasValue ? GetBoolean(value.Value) : JSValue.Null;
```

Comment:
>JSValue(bool? value) [](http://example.com/codeflow?start=36&length=20)

I wonder if we can use a generic instead.

Reply:
Updated. It's a _little_ more concise.